html/Good_practices_in_Web_Dev/good-practices-in-web-dev.md

```diff
@@ -0,0 +1,248 @@
+# Good practices in Web-Dev
+
+To make sure quality user experience, your website must be displayed correctly across all the devices. To achieve this, we need to write scalable code and give the same output for all the devices.
```

Comment:
The intro is little misleading. Best practices is used to insure that your code follow industry standard and user also doesn't miss on anything. Scalability is a different domain.

How about "Let's talk about few key points that you can take care to provide a consistent user experience across all devices and all kinds of users." You can update as you prefer.